scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -479,6 +479,22 @@
 
   (restore-offers [this pool-name offers])
 
+  (get-outstanding-synthetic-pods [this pool-name]
+    (->> (get-pods-in-pool this pool-name)
+         (add-starting-pods this)
+         (filter synthetic-pod->job-uuid)))
+
+  (set-synthetic-pods-counters [_ pool-name num-synthetic-pods]
```

Comment:
This would become
`(set-synthetic-pods-counters [cluster-name pool-name num-synthetic-pods]` and be outside of the compute cluster API.

Reply:
Same here as previous comment, I think this would break the abstraction we maintain in scheduler.clj that is agnostic to kubernetes vs mesos.